app/models/org.rb

```diff
@@ -142,6 +142,10 @@ def check_api_credentials
     end
   end
 
+  def org_admins
+    User.joins(:perms).where("users.org_id = ? AND perms.id IN (?)", self.id, [5, 6, 7, 9])
```

Comment:
using the name of the perm, although probably less fast would be more readable

Reply:
yes that's true. will change to use names